### ISSUE TITLE
How to load the parallel data to fine tune the pre-trained model "LaBSE"?

### ISSUE THREAD

**Issue by michelleqyhqyh** [UKPLab/sentence-transformers#1393]
I want to use the Class "ParallelSentencesDataset" to load my very big parallel data to fine tune the pre-trained model "LaBSE", but When I used it , how should i fill the model parameter? I mean , it seems that this  Class "ParallelSentencesDataset" need the parameters "student_model=student_model, teacher_model=teacher_model". However, I only have one model "LaBSE", how should I used this Class "ParallelSentencesDataset" ?

**Comment by nreimers** [comment]
ParallelSentencesDataset class is intended to be used for:
https://arxiv.org/abs/2004.09813

There you want to create a multilingual student from a monolingual teacher.

For normal training, see: https://www.sbert.net/docs/training/overview.html

**Comment by Oscarjia** [comment]
Thanks for the information! If I want to enhance `sentence-transformers/LaBSE` 's performance on my own multilingual data like:
```
sentence1_list = ["My first sentence", ""Mi primera frase"]
sentence2_list = ["My second sentence", "第二个局长"]
labels_list = [0.8, 0.3]
```
Can i use the way `sentence-transformers\examples\training\sts\training_stsbenchmark_continue_training.py `mentioned on [sts_continue_training](https://www.sbert.net/examples/training/sts/README.html) ？

**Comment by tomaarsen** [comment]
@Oscarjia Yes! That [training_stsbenchmark_continue_training.py](https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/sts/training_stsbenchmark_continue_training.py) training script uses CosineSimilarityLoss:
![image](https://github.com/UKPLab/sentence-transformers/assets/37621491/269ae7c1-94c8-49cd-a8c7-d9b4aaccfb29)

With `(sentence_A, sentence_B)` pairs and float similarity scores. If your data has that same format (i.e., two texts and a similarity score between 0 and 1), then you can use that script and you'd only have to change the dataset loading.

If your own multilingual data is different (e.g. positive pairs: `(sentence_A, sentence_B)` pairs without any label: this is a pretty common format), then you might want to use a different loss function instead, for example MultipleNegativesRankingLoss. This is rather similar to the [training_nli_v2.py](https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/nli/training_nli_v2.py) training script.

- Tom Aarsen

**Comment by Oscarjia** [comment]
@tomaarsen  I really appreciate your detail explanation. I also have a concern if I only load and train the exists model on the dataset of my own, will it cause forgetting catastrophic?

**Comment by tomaarsen** [comment]
Apologies for the delay, I went on a short vacation.
No, you should not get any catastrophic forgetting: Sentence Transformer models can be reasonably finetuned on new data without forgetting previously learned data. 

- Tom Aarsen

**Comment by Oscarjia** [comment]
Cool, thank you for your explanation, that is fantastic! 
But i also curious why it can also keep the learned weights while finetuning on new datasets?

**Comment by tomaarsen** [comment]
Apologies for the confusion, it won't keep exactly the original weights: they will be modified to get better for your task. The model will generally not degrade in performance very much on the original task, i.e. it generally won't catastrophically forget when you finetune, despite the weights being updated.

- Tom Aarsen

**Comment by Oscarjia** [comment]
@tomaarsen  Hi Tom, 

i use the [training_stsbenchmark_continue_training.py](https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/sts/training_stsbenchmark_continue_training.py) to finetuing the model on my own datasets, i found there are runs log directory under the output_dir,  the log show 

  `"do_train": false ` But why the "do_train" value is not `true`?

Here is more log:
```
<A0>
^Qargs/text_summaryB<80>^G^R^D^R^B^H^AB<F5>^_{
  "output_dir": "output/custss7",
  "overwrite_output_dir": false,
  "do_train": false,
  "do_eval": true,
  "do_predict": false,
  "eval_strategy": "steps",
  "prediction_loss_only": true,
  "per_device_train_batch_size": 120,
  "per_device_eval_batch_size": 120,
  "per_gpu_train_batch_size": null,
  "per_gpu_eval_batch_size": null,
  "gradient_accumulation_steps": 1,
  "eval_accumulation_steps": null,
  "eval_delay": 0,
  "learning_rate": 2e-05,
  "weight_decay": 0.0,
  "adam_beta1": 0.9,
  "adam_beta2": 0.999,

```
